### ISSUE TITLE
contrib-tools: cmd/cueckoo: track original commit we import

### ISSUE THREAD

**Issue by myitcv** [cue-lang/cue#4309]
Right now, an import performs a couple of commit rewrites (altering the commit message only) which by definition means we find ourselves with a different commit hash in Gerrit to that sitting in GitHub. We cannot avoid this. However, we should track somewhere the original commit hash for reference, but also to help debug situations where there is a race condition on someone force pushing a commit to a PR at around the time the import is being performed (this has happened). 

One additional/alternative measure here would be to have `cmd/cueckoo importpr` write a message to the PR at the time the PR is imported, including the commit hash that was imported as well as the resulting change ID. This does not totally prevent a race condition, but would greatly help the debugging. 

cc @mvdan FYI.

**Comment by myitcv** [comment]
https://github.com/cue-sh/tools/commit/fa87e8579a2d5a30cf37dff11a549d67b7f7ca41 adds the short commit hash to "Closes" for `importpr`. Will leave this open in case we feel like adding the writing of a PR comment at the point of import.

**Comment by mvdan** [comment]
> Will leave this open in case we feel like adding the writing of a PR comment at the point of import.

I don't think this is necessary. I import less than two PRs per month, so writing a comment for the person barely takes any time. Not worth automating.
